pathfinder/model/channel_view.py

```diff
@@ -36,14 +36,20 @@ def __init__(
         self._capacity = deposit
         self.state = ChannelView.State.OPEN
         self.channel_id = channel_id
+        self.balance_proof_nonce = -1
```

Comment:
Balance proofs can not start with the number zero, that is the default value for uninitialized memory on the evm. Using `-1` as a default allows would accept a balance proof with a nonce of `0` as valid, which is wrong. The correct default is `0`.

Reply:
\*\*The more you know\*\*